contrib/ctags_profile.awk

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env awk -f
```

Comment:
maybe place a new directory: ./contrib/ctags - since the could be multiple ctags files for macros translations.
OR - rename this file to something more generic that would not be only for profile functions

Reply:
fixed option 2